api/serializers/__init__.py

```diff
@@ -147,3 +147,13 @@ class Meta:
         model = User
         fields = ["email"]
         read_only_fields = ["email"]
+
+
+class WebcompatIssueSerializer(serializers.Serializer):
+    issue_on_domain = serializers.URLField(
+        max_length=200, min_length=None, allow_blank=False
+    )
+    email_mask_not_accepted = serializers.BooleanField(required=False, default=False)
+    add_on_visual_issue = serializers.BooleanField(required=False, default=False)
+    email_not_received = serializers.BooleanField(required=False, default=False)
```

Comment:
question (non-blocking): Does the UI allow someone to provide any, all, or none of these field values? I'm wondering if these should be 3 boolean fields, or a single `CharField` with a finite number of options?